packages/wpcom-proxy-request/examples/webpack.config.js

```diff
@@ -1,12 +1,14 @@
+const path = require( 'path' );
+
 module.exports = {
-	entry: __dirname + '/../build/index.js',
+	entry: path.join( __dirname, '/../build/index.js' ),
```

Comment:
The leading slashes are not needed and could suggest that the path is absolute. Is this an application of the rule's autofix, or did you make the changes manually?

Reply:
I wrote a regexp to autofix it (the rule doesn't provide any). I can change it to remove the initial `/`